react/components/SearchBar/SearchBar.tsx

```diff
@@ -21,6 +21,7 @@ import AutocompleteResults from '../../AutocompleteResults'
 import type { Props as AutocompleteResultsProps } from '../../AutocompleteResults'
 import AutocompleteInput from './AutocompleteInput'
 import { useSearchBarCssHandles } from './SearchBarCssHandles'
+import { usePixel } from 'vtex.pixel-manager'
```

Comment:
Running `yarn lint` at the root of the repo reports an error here:
```
react/components/SearchBar/SearchBar.tsx
   23:1   error    There should be at least one empty line between import groups                import/order
   24:1   error    `vtex.pixel-manager` import should occur before import of `./SearchBar.css`  import/order
```
Moving this to line 18 seems to fix it.